plugins/reolink/src/reolink-api.ts

```diff
@@ -201,16 +201,53 @@ export class ReolinkCameraClient {
         const url = new URL(`http://${this.host}/api.cgi`);
         const params = url.searchParams;
         params.set('cmd', 'GetDevInfo');
-        const response = await this.requestWithLogin({
+        let response = await this.requestWithLogin({
             url,
             responseType: 'json',
         });
-        const error = response.body?.[0]?.error;
+        let error = response.body?.[0]?.error;
         if (error) {
             this.console.error('error during call to getDeviceInfo', error);
             throw new Error('error during call to getDeviceInfo');
         }
-        return response.body?.[0]?.value?.DevInfo;
+
+        const deviceInfo: DevInfo = await response.body?.[0]?.value?.DevInfo;
+
+        // Will need to check if it's valid for NVR and NVR_WIFI
+        if (['HOMEHUB'].includes(deviceInfo.exactType)) {
```

Comment:
invert this logic so if it doesn't include, it immediately returns deviceInfo. Remaining code will be only for reolink home hub logic.

Reply:
done!